app/src/main/java/com/adam/aslfms/service/Scrobbler.java

```diff
@@ -304,16 +304,21 @@ public void scrobbleCommit(HandshakeResult hInfo, Track[] tracks)
                 Log.d(TAG, response);
                 if (resCode == 401) {
                     settings.setListenBrainzToken(netApp, "");
-                    throw new BadSessionException("Now Playing failed because of badsession");
+                    throw new BadSessionException("Listenbrainz submission failed because of bad token.");

```

Comment:
"ListenBrainz", not "Listenbrainz", please. I already made a PR to fix all instances of the lowercase 'b' once, please don't introduce new ones. 😢